### ISSUE TITLE
Black screen after repeating action in 'MineRLNavigateDense-v0' enviroment

### ISSUE THREAD

**Issue by dk000000000** [minerllabs/minerl#52]
It seems that 'MineRLNavigateDense-v0' enviroment will return observation (64,64,3) with all zeros after repeating same action few time, action causing the problem can either be random sampled using `env.action_space.sample()` or created using `OrderedDict([('attack', 1),
             ('back', 0),
             ('camera', [  0, 0 ]),
             ('forward', 1),
             ('jump', 1),
             ('left', 0),
             ('place', 0),
             ('right', 0),
             ('sneak', 0),
             ('sprint', 0)])`
Code I used:
```
import gym
import minerl
import numpy as np

env = gym.make('MineRLNavigateDense-v0')
env.reset()

action = env.action_space.sample()
obs, reward, done, _ = env.step(action);print(np.sum(obs['pov']))
obs, reward, done, _ = env.step(action);print(np.sum(obs['pov']))
# some times occur even after three repeat
obs, reward, done, _ = env.step(action);print(np.sum(obs['pov']))

env.reset()

newaction = OrderedDict([('attack', 1),
             ('back', 0),
             ('camera', [  0, 0 ]),
             ('forward', 1),
             ('jump', 1),
             ('left', 0),
             ('place', 0),
             ('right', 0),
             ('sneak', 0),
             ('sprint', 0)])

obs, reward, done, _ = env.step(newaction);print(np.sum(obs['pov']))
obs, reward, done, _ = env.step(newaction);print(np.sum(obs['pov']))
# still happens even after reset 
obs, reward, done, _ = env.step(newaction);print(np.sum(obs['pov']))
```

**Comment by alstar8** [comment]
Have the same error with MineRLObtainDiamond-v0 environment

And after a few steps obs['pov'] start returning black screen (all values of it are 0)
The first steps before the black screen are normal and even after the black screen environment continues to act normal and things like inventory are 

[url=https://ibb.co/mRP5C4v][img]https://i.ibb.co/104GfTs/black-screen.png[/img][/url]

I use anaconda with python 3.6 and Windows 10

**Comment by MadcowD** [comment]
This is a *glaring issue*! Working on it now :)

**Comment by MadcowD** [comment]
I'm unable to reproduce this directly drom the code attached; this must be a race condition. Investigating further.

**Comment by MadcowD** [comment]
@dk000000000 Are you on windows?

**Comment by dk000000000** [comment]
yes, I install it with pip

**Comment by MadcowD** [comment]
@dk000000000  I believe this is fixed with `minerl` > 0.1.10. Please upgrade to see if this fixes your issue
```
pip3 install --upgrade minerl
```

**Comment by dk000000000** [comment]
Still give 0 after updating to 0.1.11, ubuntu version works, guess I will just switch to ubuntu enviroment
